customcommands/bot.go

```diff
@@ -156,7 +156,7 @@ var cmdEvalCommand = &commands.YAGCommand{
 		ctx := templates.NewContext(data.GuildData.GS, channel, data.GuildData.MS)
 		ctx.IsExecedByEvalCC = true
 
-		code := common.ParseCodeblock(data.Args[0].Str())
+		code := common.ParseCodeblock(data.TraditionalTriggerData.MessageStrippedPrefix)
```

Comment:
This change absolutely merits a comment in the code itself IMO, not just some exposition in the PR description.

Reply:
How's this?

Reply:
How's this?